README.md

```diff
@@ -1 +1,37 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello Microverse
+
+> This is the first HTML and CSS project at Microverse.
+
+
```

Comment:
- [ ] Please update your Readme, currently some headings are missing such as  

```
##  Getting Started 
```
- [ ] Your README looks great, good job 👍🏻  however  it will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes `git clone URL` , `cd project_name` etc please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.